.travis.yml

```diff
@@ -166,6 +166,33 @@ jobs:
         - cd $TRAVIS_BUILD_DIR/../metis-4.0;
           mv libmetis.a ..; rm -rf *; mv ../libmetis.a .
 
+    - os: linux
+      compiler: gcc
+      name: "Linux: Parallel (cmake)"
+      addons:
+        apt:
+          packages:
+            - mpich
+            - libmpich-dev
+      env: MPI=YES
+      script:
+        - cd ${TRAVIS_BUILD_DIR}
+        - mkdir ${TRAVIS_BUILD_DIR}/build
+        - cd ${TRAVIS_BUILD_DIR}/build
+        - ls ${TRAVIS_BUILD_DIR}/../metis-4.0
```

Comment:
Full disclosure: I've never worked with anything but a very simple Travis script.  Is this `ls` necessary/intended here or is it leftover debugging code?

Reply:
This looks like a debug leftover, yes.

Reply:
Once it is removed, I think this should be merged.